registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/code/RegistrationTransactionTypeCode.java

```diff
@@ -1,47 +1,56 @@
-package io.mosip.registration.processor.core.code;
-
-/**
- * The Enum RegistrationTransactionTypeCode.
- */
-public enum RegistrationTransactionTypeCode {
-
-	/** The packet receiver. */
-	PACKET_RECEIVER,
-	/** The virus scan. */
-	VIRUS_SCAN,
-	/** The upload packet. */
-	UPLOAD_PACKET,
-	/** The validate packet. */
-	VALIDATE_PACKET,
-	/** The packet classification. */
-	PACKET_CLASSIFICATION,
-	/** The osi validate. */
-	OSI_VALIDATE,
-	/** The external integration. */
-	EXTERNAL_INTEGRATION,
-	/** The demographic verification. */
-	DEMOGRAPHIC_VERIFICATION,
-	/** The manual varification. */
-	MANUAL_VERIFICATION,
-	/** The biographic verification. */
-	BIOGRAPHIC_VERIFICATION,
-	/** The uin generator. */
-	UIN_GENERATOR,
-	/** The notification. */
-	NOTIFICATION,
-	/** The print service. */
-	PRINT_SERVICE,
-	/** The print postal service. */
-	PRINT_POSTAL_SERVICE,
-    /** The print stage. */
-    PRINT,
-	/** The reprocess. */
-	PACKET_REPROCESS,
-	/** The Quality Check*/
-	QUALITY_CHECK,
-	/** The biometric Authentication*/
-	BIOMETRIC_AUTHENTICATION,
-	/** The securezone notification stage */
-	SECUREZONE_NOTIFICATION
-
-}
+package io.mosip.registration.processor.core.code;
+
+// TODO: Auto-generated Javadoc
+/**
+ * The Enum RegistrationTransactionTypeCode.
+ */
+public enum RegistrationTransactionTypeCode {
+
+	/** The packet receiver. */
+	PACKET_RECEIVER,
+	/** The virus scan. */
+	VIRUS_SCAN,
+	/** The upload packet. */
+	UPLOAD_PACKET,
+	/** The validate packet. */
+	VALIDATE_PACKET,
+	/** The packet classification. */
+	PACKET_CLASSIFICATION,
+	/** The osi validate. */
+	OSI_VALIDATE,
+	/** The external integration. */
+	EXTERNAL_INTEGRATION,
+	/** The demographic verification. */
+	DEMOGRAPHIC_VERIFICATION,
+	/** The manual varification. */
+	MANUAL_VERIFICATION,
+	/** The biographic verification. */
+	BIOGRAPHIC_VERIFICATION,
+	/** The uin generator. */
+	UIN_GENERATOR,
+	/** The notification. */
+	NOTIFICATION,
+	/** The print service. */
+	PRINT_SERVICE,
+	/** The print postal service. */
+	PRINT_POSTAL_SERVICE,
+    /** The print stage. */
+    PRINT,
+	/** The reprocess. */
+	PACKET_REPROCESS,
+
+	/** The Quality Check. */
+	QUALITY_CHECK,
+
+	/** The biometric Authentication. */
+	BIOMETRIC_AUTHENTICATION,
+
+	/** The securezone notification stage. */
+	SECUREZONE_NOTIFICATION,
+
+	/** The workflow event update. */
+	WORKFLOW_EVENT_UPDATE,

```

Comment:
WORKFLOW_EVENT_UPDATE and WORKFLOW_ACTION_SERVICE is not a transaction type

Reply:
updated to WORKFLOW_PAUSE and WORKFLOW_RESUME